medallion/test/test_backends.py

```diff
@@ -390,90 +383,67 @@ def test_get_objects_version(backend):
         # if obj["id"] == "malware--c0931cc6-c75e-47e5-9036-78fabc95d4ec":
         #    assert obj["modified"] == "2018-02-23T18:30:00.000Z"
 
-    r = backend.client.get(
-        test.GET_OBJECTS_EP + "?match[version]=all",
-        headers=backend.headers,
-    )
 
-    assert r.status_code == 200
-    assert r.content_type == MEDIA_TYPE_TAXII_V21
-    objs = r.json
-    assert objs['more'] is False
+def test_objects_version_match_all(backend):
+    objs = get_objects_by_version(backend, "?match[version]=all")
     assert len(objs['objects']) == 7
 
 
-def test_get_objects_spec_version(backend):
+# following is new test_get_objects_spec_version
+def get_objects_spec_version(backend, getstr, length):
```

Comment:
Might consider renaming `length` to `num_objects` or something similar (length of what?).